MediaBrowser.Model/Querying/NextUpQuery.cs

```diff
@@ -13,6 +13,7 @@ public NextUpQuery()
             EnableImageTypes = Array.Empty<ImageType>();
             EnableTotalRecordCount = true;
             DisableFirstEpisode = false;
+            NextUpDateCutoff = new DateTime(0001, 01, 01);
```

Comment:
```suggestion
            NextUpDateCutoff = null;
```

Also, `DateTime.MinValue` is preferred over this.